source/_integrations/hydrawise.markdown

```diff
@@ -77,6 +77,11 @@ monitored_conditions:
 The Hydrawise API removed the ability to read the rain sensor status. Therefore it is no longer supported by the Hydrawise integration to Home Assistant.
 </div>
 
+<div class='note warning'>
+The Hydrawise API uses rate limiting and might throw errors in case the scan_interval is too low or too many manual service calls are triggered:
+The limit is 3 calls to start/stop/suspend a zone per 30 seconds and an an additional limit across the entire API of 30 calls in a 5 minute period per user.
```

Comment:
```suggestion

The Hydrawise API uses rate limiting and might throw errors in case the `scan_interval` is too low or too many manual service calls are triggered:
The limit is 3 calls to start/stop/suspend a zone per 30 seconds and an additional limit across the entire API of 30 calls in a 5-minute period per user.

```